ss58-registry.json

```diff
@@ -550,6 +550,24 @@
       "standardAccount": "*25519",
       "website": "https://sora.org"
     },
+    {
+      "prefix": 71,
+      "network": "3dp",
+      "displayName": "3DP network",
+      "symbols": ["3DP"],
+      "decimals": [12],
+      "standardAccount": "*25519",
+      "website": "https://3dpass.org"
+    },
+    {
+      "prefix": 72,
+      "network": "3dpt",
+      "displayName": "3DP test network",
+      "symbols": ["3DPt"],
+      "decimals": [12],
+      "standardAccount": "*25519",
+      "website": "https://3dpass.org"
+    },
```

Comment:
```suggestion
```

Test networks don't get their own ss58 prefix.

Reply:
Hello @bkchr. 3DPt testnet represents a canary network already running now. We will keep working both networks in live after we start the mainnet. So, 72 is not a conventional testnet, we just call it "Testnet". https://telemetry.polkadot.io/#list/0xd07b37f1046f061b5b17cef78fd171f1a9386d2cd8f465312877a7cbe5e301e5  For that reason, we need both 71 and 72 prefixes to be added.

Reply:
Okay ty for the explanation